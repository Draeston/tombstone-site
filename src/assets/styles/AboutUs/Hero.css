@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Arabic:wght@200;300;400&display=swap');

.hero {
    background: url("../../images/AboutUs/hero.jpg") no-repeat;
    background-position-y: 80%;
    background-size: cover;
    position: relative;
    padding: 50px;

    min-height: 300px;
}

.hero-text {
    font-family: 'IBM Plex Sans Arabic', sans-serif;
    font-size: 24px;
    color: white;
    text-align: center;
}
