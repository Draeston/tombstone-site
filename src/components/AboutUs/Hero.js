import styles from  "../../assets/styles/AboutUs/Hero.module.css"

export function Hero () {
    return (
        <div className={styles.hero}>
            <p className={styles.heroText}>
                Some text here
            </p>
        </div>
    );
}
