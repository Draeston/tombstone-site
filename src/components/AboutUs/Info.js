import styles from "../../assets/styles/AboutUs/Info.module.css"

export function Info () {
    return (
        <div className={styles.info}>
            <p className={styles.infoText}>
                Some text here
            </p>
        </div>
    )
}
