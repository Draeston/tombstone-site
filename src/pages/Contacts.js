import { Card } from "../components/Contacts/Card";

export function Contacts(props) {
  const listItems = props.contacts.map((contact) => (
    <Card img={contact.img} name={contact.name} telegram={contact.telegram} />
  ));
  return <div>{listItems}</div>;
}
