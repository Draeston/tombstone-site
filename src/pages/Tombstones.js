import { Card } from "../components/Tombstones/Card";
import { useEffect, useState } from "react";
import "../assets/styles/Tombstones/Tombstones.css";

export function Tombstones() {
  const [list, setList] = useState([]);

  useEffect(() => {
    fetch("/api/tombstones")
      .then((response) => response.json())
      .then((data) =>
        setList(
          data.tombstones.map((item) => (
            <Card
              key={item.id}
              img={item.img}
              title={item.title}
              price={item.price}
              id={item.id}
            />
          ))
        )
      );
  });

  return <div className="wrapper">{list}</div>;
}
