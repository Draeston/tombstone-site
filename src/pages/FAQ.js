import "../assets/styles/FAQ/FAQ.css";
import { QA } from "../components/FAQ/QA";
import Hero from "../components/FAQ/Hero";

export function FAQ(props) {
  const listItems = props.qas.map((item) => (
    <QA question={item.question} answer={item.answer} key={item.question} />
  ));
  return (
    <div className="faq">
      <Hero />
      {listItems}
    </div>
  );
}
