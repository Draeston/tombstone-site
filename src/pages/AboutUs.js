import {Hero} from "../components/AboutUs/Hero";
import {Info} from "../components/AboutUs/Info";

export function AboutUs () {
    return (
        <div>
            <Hero />
            <Info />
        </div>
    )
}
