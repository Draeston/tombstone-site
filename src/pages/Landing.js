// background images will be changed later
import { Hero } from "../components/Landing/Hero";
import { Info } from "../components/Landing/Info";
import MakeOrder from "../components/Landing/MakeOrder";

export function Landing() {
  const infoText = [];
  infoText.push(
    "We are glad to help the customers to decide and design proper and cheap product that will delivered their waits as fast as possible and to avoid any difficulties that can occur."
  );
  infoText.push(
    "Our tombstones are really of high quality and we will make sure it will not be destroyed in 5-10 years"
  );
  infoText.push(
    "We will make sure that your buried will rest in the ground calmly"
  );
  return (
    <div>
      <Hero />
      <Info text={infoText[0]} image="1.png" isReverted={false} />
      <Info text={infoText[1]} image="2.png" />
      <Info text={infoText[2]} image="3.png" isReverted={false} />
      <MakeOrder />
    </div>
  );
}
