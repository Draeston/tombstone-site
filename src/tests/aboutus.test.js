import React from "react";
import renderer from "react-test-renderer";
import { AboutUs } from "../pages/AboutUs";

test("snapshot about us", () => {
  const tree = renderer
    .create(<AboutUs />)
    .toJSON()
  expect(tree).toMatchSnapshot();
})
