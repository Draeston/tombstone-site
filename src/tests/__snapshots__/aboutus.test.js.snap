// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot about us 1`] = `
<div>
  <div
    className="hero"
  >
    <p
      className="heroText"
    >
      We decided to help our customers and support them durring the whole process of such ridiculous situation
    </p>
  </div>
  <div
    className="info"
  >
    <p
      className="infoText"
    >
      We are glad to help the customers to decide and design proper and cheap product that will delivered their waits as fast as possible and to avoid any difficulties that can occur.
    </p>
  </div>
</div>
`;
